go/keymanager/api/api.go

```diff
@@ -130,6 +131,39 @@ type Genesis struct {
 	Statuses []*Status `json:"statuses,omitempty"`
 }
 
+// SanityCheck does basic sanity checking on the genesis state.
+func (g *Genesis) SanityCheck() error {
+	for _, status := range g.Statuses {
+		// Verify key manager runtime ID.
+		if !status.ID.IsValid() {
+			return fmt.Errorf("keymanager: sanity check failed: key manager runtime ID %s is invalid", status.ID.String())
+		}
+
+		// Verify currently active key manager node IDs.
+		for _, node := range status.Nodes {
+			if !node.IsValid() {
+				return fmt.Errorf("keymanager: sanity check failed: key manager node ID %s is invalid", node.String())
+			}
+		}
+
+		// Verify SGX policy signatures if the policy exists.
+		if status.Policy != nil {
```

Comment:
Hm, a key manager without a policy is largely nonsensical, but we can't tell if enforcement is disabled easily. :(